docs/.redirects/redirects.json

```diff
@@ -466,4 +472,4 @@
     "tutorials/porting-tutorial": "pytorch-mnist-tutorial.html",
     "tutorials/quick-start": "quickstart-mdldev.html",
     "tutorials/pytorch-mnist-local-qs": "../get-started/webui-qs.html"
-}
\ No newline at end of file
+}
```

Comment:
Thank you for adding missing newlines back.

/Grumbles at whatever web editor some people use which keeps pruning one byte from files.